src/YiiElasticSearch/MultiSearchDataProvider.php

```diff
@@ -0,0 +1,133 @@
+<?php
+namespace YiiElasticSearch;
+
+/**
+ * A Data provider for searching into multiple indices
+ */
+class MultiSearchDataProvider extends DataProvider
+{
+    public $connID = 'elasticSearch';
```

Comment:
@mikehaertl True but in ES you can't search in multi instances i think, so there is no use actually using the models connection ID. Both approaches could result unexpected results.

However the optimal solution here could be to pass the connection ID in the class constructor, so if any of the passed models is using another then it MUST throw an exception.